### ISSUE TITLE
Reported issue with md-select in conjunction with JWAS screen reader

### ISSUE THREAD

**Issue by christophe-g** [material-components/material-web#5136]
### What is affected?

Accessibility, Component

### Description

We have received reports from users that `md-select` component is not behaving as expected when used with JAWS screen reader. 

Since "Support more screen readers" is in the 2023 Q4 roadmap, I also report here, although I think it is more of  a JAWS bug (https://github.com/FreedomScientific/standards-support/issues/776).

Note that it is working correctly with the screen reader we use for testing (Orca on Linux).

### Steps to reproduce:
https://lit.dev/playground/#gist=8266d1cf3ac2739e43a64cfa77bc02bc

### Expected result
- Options are announced when they receive focus (after the listbox menu is opened). JAWS should announce: "blank", "apple", "orange", "banana" as for the native select component.
- The selection menu closes when a user select a value. (enter or space key)

### Actual result
The reported behavior is that:

- Options are not announced when they receive focus
- The selection menu stays open once a user has selected a value.
- The selection menu needs to be closed manually (by pressing the escape key for instance)

### Workaround

We did not receive such reports when our select component was based on MD2. So one possible workaround will be to downgrade our select component back to MD2.

### Is this a regression?

No or unsure. This never worked, or I haven't tried before.

### Affected versions

Possibly working under MD2, reported to fail with MD3 1.0.1

### Browser/OS/Node environment

Jaws 2023, windows, Chrome latest

**Comment by asyncLiz** [comment]
Thank you SO MUCH for this bug! We rarely get SR bugs, and this will be super helpful for the next two quarters.

**Comment by christophe-g** [comment]
> We rarely get SR bugs, and this will be super helpful for the next two quarters.

Thanks @asyncLiz - we might receive more bug/issue reports in the future as we doing continuous accessibility user testing for our app ([accessiblesurveys](https://accessiblesurveys.com)).

On this particular one, JAWS cannot reproduce. I am trying to arrange a session with the user reporting the issue to better understand what might be happening.
